ggshield/utils/files.py

```diff
@@ -35,9 +44,7 @@ def get_filepaths(
     paths: List[Path],
     exclusion_regexes: Set[re.Pattern],
     recursive: bool,
```

Comment:
The enum replaces `recursive` too: `FILES_ONLY` is the equivalent of `recursive=False`.